README.md

```diff
@@ -42,6 +42,22 @@ This TodoList app uses Azure DocumentDB spring boot starter and AngularJS to int
         <version>${azure.spring.boot.starter.version}</version>^M
     </dependency>
     ```
+    
+* Customize Angular JS
+In `app.js`, make following changes. The client leverages Azure AD library for JS to handle AAD authentication in single page application. The following snippet of code configures adal provider for your registered app.
```

Comment:
same as in aad-start branch. pls help to specify full path of app.js file.

and pls add detail steps on how to get tenant and client id. and what they are